exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/FragmentWrapper.java

```diff
@@ -35,6 +38,10 @@
 public class FragmentWrapper {
   private final MajorFragmentProfile major;
   private final long start;
+  private final Locale currentLocale = Locale.getDefault();
+  private final String pattern = "dd-MMM-yyyy HH:mm:ss";
+  private final SimpleDateFormat simpleDateFormat = new SimpleDateFormat(
```

Comment:
The following is the same:
```
new SimpleDateFormat(pattern);
```

Note that the locale info *does not* adjust the time zone. Local time is quite clunky with the original Java classes; we need to use the Joda classes to better handle local date/times.